### PR TITLE
Release/public v2

### PR DESCRIPTION
**Pull request by natalie-perlin**
HPC-stack libraries built for the SRW app: ufs-srweather-app release/public-v2 , 
using GNU compilers (GNU10.2 or 9.2) and openmpi, on Orion, Gaea, Hera, Jet. 

Notes: 
**Gaea** 
* library geos is downloaded from github, repo: https://github.com/libgeos/geos.git

**Hera** 
* building anaconda downloaded from another source (not NOAA)
* turned off building: madis , boost

**Jet** 
* turned off building: bufr 
* still problem with downloadig wgib2.
 (possible solution: download from github? - need to check and possibly modify installation steps, including wgrib2 - grib2 -grib_utils directory name differences)
branch:  feature/v2.0.8 repo: **https://github.com/NOAA-EMC/NCEPLIBS-wgrib2.git**
 
 Use a variable STACK_esmf_os on Gaea, to set ESMF_OS="Linux"

### CODE REVIEW

**Comment by natalie-perlin** [comment]
Added configuration of stack modules built on MacOS with Intel/x86_64 and M1/arm64. All are built with GNU 11.2.0 (Homebrew 11.3.0 formula) and openmpi/4.1.2. There is no difference in stack/stack_macos_<...>.yaml files at the moment.

**Comment by natalie-perlin** [comment]
Issue of certain software downloads blocked by the NOAA firewall on some of the systems. Solution:
In some cases, other alternatives were found (proj, anaconda). In some cases (madis, wgrib2, boost libraris) the tar files were downloaded as raw files from a GitHub public repo created specifically for that purpose https://github.com/natalie-perlin/HPC-stack-NOAA-blocked-downloads.

Location of successfull builds of the stack for the release/public-v2, with GNU compiler and openmpi:

On Orion:  /work/noaa/epic-ps/nperlin/HPC-stack/hpc-stack/ ,
/work/noaa/epic-ps/nperlin/HPC-stack/install_1.2.0_gnu10

On Jet: 
/home/Natalie.Perlin/HPC-stack/hpc-stack/  , 
/home/Natalie.Perlin/HPC-stack/install-1.2.0_gnu9.2

On Hera:  
/scratch1/NCEPDEV/stmp2/Natalie.Perlin/HPC-stack/hpc-stack ,
/scratch1/NCEPDEV/stmp2/Natalie.Perlin/HPC-stack/install-1.2.0_gnu9.2

On Gaea:
/ncrc/home2/Natalie.Perlin/HPC-stack/hpc-stack,
/ncrc/home2/Natalie.Perlin/HPC-stack/install_gnu10.3

All the ./pkg/ directories with the builds and downloads have been cleaned after the successful builds, but the log files remained in the build directories.

 



 



 



To download and test the repository:
git clone -b release/public-v2 https://github.com/NOAA-EPIC/hpc-stack.git hpc-stack

**Comment by kgerheiser** [comment]
I'm fine with this, but I think the branch should be `release/srw-public-v2`. There are other applications that `release/public-v2` could apply to.

File: stack/stack_orion_gnu.yaml [hunk 1]
```diff
@@ -0,0 +1,358 @@
...
+  build: YES
+  version: 3.6.1
+  repo: ecmwf
+
+eckit:
+  build: YES
+  version: 1.16.0
+  repo: ecmwf
+
+fckit:
```
Comment (kgerheiser) [inline stack/stack_orion_gnu.yaml:298]: And if this is just for srw-v2, I think we should disable these JEDI builds that aren't sepcifically needed.

Comment (natalie-perlin) [reply stack/stack_orion_gnu.yaml:298]: Which libraries belong to Jedi?

Comment (natalie-perlin) [reply stack/stack_orion_gnu.yaml:298]: I will update the branch name or will create another one `release/srw-public-v2`

**Comment by kgerheiser** [comment]
Ok, closing this.

**Pull request closed without merging**